Cartridges/int_checkoutcom_sfra/cartridge/scripts/helpers/eventsHelper.js

```diff
@@ -126,7 +126,7 @@ var eventsHelper = {
             paymentInstrument.paymentTransaction.setType(PaymentTransaction.TYPE_CAPTURE);
 
             setPaymentStatus(order);
-            order.setConfirmationStatus(1);
+            order.setConfirmationStatus(2);
```

Comment:
What are these numbers? It does not mean anything if someone from outside look at this part

Reply:
Maybe a constant which also indicate the meaning would clarify